vms/registration/views.py

```diff
@@ -31,6 +31,9 @@ class AdministratorSignupView(TemplateView):
     phone_error = False
 
     def get(self, request):
+        if request.user.is_authenticated():
+            if not hasattr(request.user, 'administrator'):
```

Comment:
@mayank-agarwal-96 Well... I thought as it is only needed for this view, i would just protect it by adding code in both the functions.